services/skus/controllers.go

```diff
@@ -588,6 +589,47 @@ func CreateOrderCreds(svc *Service) handlers.AppHandler {
 	}
 }
 
+// deleteItemCreds handles requests for deleting credentials for an item.
+func deleteItemCreds(svc *Service) handlers.AppHandler {
+	return func(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+		ctx := r.Context()
+		lg := logging.Logger(ctx, "skus.deleteItemCreds")
+
+		orderID := &inputs.ID{}
+		if err := inputs.DecodeAndValidateString(ctx, orderID, chi.URLParamFromCtx(ctx, "orderID")); err != nil {
```

Comment:
question: are we not trying to move away from this approach of using the `DecodeAndValidateString` given it adds an extra layer of abstraction which seems unnecessary?

Reply:
addressed: https://github.com/brave-intl/bat-go/pull/2281/commits/a40fb6cdf80029cc56e66e28a0ede7d70f191587